src/gf.c

```diff
@@ -2795,7 +2795,8 @@ static void record_dispatch_statement(jl_method_instance_t *mi)
             s_dispatch = (JL_STREAM*) &f_dispatch;
         }
     }
-    if (!jl_has_free_typevars(mi->specTypes)) {
+    // NOTE: Sometimes the specType is just `Tuple`, which is not useful to print.
+    if (!jl_has_free_typevars(mi->specTypes) && (jl_datatype_t*)mi->specTypes != jl_tuple_type) {
```

Comment:
I'm not sure what this is about, but without this, i was seeing several lines like this:
```
precompile(Tuple)
precompile(Tuple)
precompile(Tuple)
```

Reply:
That is the signature for builtin functions, which are dispatched via pre-populated cache entries.

Reply:
Aha, thanks. So skipping them is right? I'll just improve the comment.